<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bootie</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="writeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="usbDevicesGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>USB Device:</string>
    </property>
    <widget class="QComboBox" name="devicesListComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deviceInfoButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/images/info_black_36dp.svg</normaloff>:/icons/images/info_black_36dp.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshDevicesListButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/images/refresh_black_36dp.svg</normaloff>:/icons/images/refresh_black_36dp.svg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>340</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="imageSelectGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>340</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Image:</string>
    </property>
    <widget class="QLineEdit" name="imagePath">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseImageButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/icons/images/folder_black_48dp.svg</normaloff>:/icons/images/folder_black_48dp.svg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>340</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QGroupBox" name="writeOptionsGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Write options:</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="systemComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Windows</string>
     </property>
     <item>
      <property name="text">
       <string>Windows</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>3</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="biosRadioButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BIOS</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="uefiRadioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>UEFI</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>340</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
